README.md

```diff
@@ -190,13 +177,42 @@ Then you just have to invoke `swiftgen config run`, or even just `swiftgen` for
 
 To learn more about the configuration file — its more detailed syntax and possibilities, how to pass custom parameters, using `swiftgen config lint` to validate it, how to use alternate config files, and other tips — [see the dedicated documentation](Documentation/ConfigFile.md).
 
+There are also more subcommands not related to generating code but more oriented for help and other tasks, namely:
+
+* The `swiftgen templates` subcommands helps you print, duplicate, find and manage templates (both bundled and custom)
+* You can use `--help` on `swiftgen` or one of its subcommand to see the detailed usage.
+
+
+### Directly invoking a subcommand
+
+While we no longer recommend it for performance reasons (if you have multiple outputs), you may prefer to not use a configuration file. You can directly invoke the available subcommands to parse various resource types:
```

Comment:
> While we highly recommend to use a configuration file for performance reasons (especially if you have multiple outputs, but also because it's more flexible), it's also possible to directly invoke the available subcommands to parse various resource types:

[…] list […]

One rare cases where this might be useful — as opposed to using a config file — is if you are working on a custom template and want to test it quickly on each iteration/version of your custom template until you're happy with it_